CMakeLists.txt

```diff
@@ -91,7 +92,7 @@ target_link_libraries(py_finger_types PRIVATE ${catkin_LIBRARIES})
 set_target_properties(py_finger_types PROPERTIES
     LIBRARY_OUTPUT_DIRECTORY ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_PYTHON_DESTINATION})
 install(TARGETS py_finger_types DESTINATION ${CATKIN_PACKAGE_PYTHON_DESTINATION})
-
+# one joint #
```

Comment:
Please keep an empty line between the blocks for different robots, that makes it a bit easier to read and handle.